public/css/userGretting.module.css

```diff
@@ -0,0 +1,36 @@
+.greetingWrapper {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  gap: 10px;
+
+  & button {
+    font-size: 20px;
+    border: none;
+    transition: 100ms;
+    background-color: rgba(255, 255, 255, 0);
+    color: #fff;
```

Comment:
`#fff` is white, didnt you set other colors with their names?

Reply:
i only use hex codes and sometimes 'rgba()' for the opacity.

Reply:
well damn